.github/workflows/swift.yml

```
@@ -14,6 +14,9 @@ jobs:
     strategy:
       matrix:
         include:
+          - os: macos-14-arm64
```

```suggestion
          - os: macos-14
```
No GitHub-hosted runner matches `macos-14-arm64` query. Use `macos-14` instead.

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories